### PR TITLE
/api/stream for non-admin return only own streams

### PR DESCRIPTION
**Pull request by darkdarkdragon**
**What does this pull request do? Explain your changes. (required)**
/api/stream for non-admin return only own streams

**Specific updates (required)**

## -

- **How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by iameli — CHANGES_REQUESTED** [review]
Minor change.

**Comment by darkdarkdragon** [comment]
@iameli Github's interface doesn't show any suggested change 😞

**Comment by wohlner** [comment]
Can I test this with a `curl` command or something in my terminal before it goes to stage? Or do I need to wait until then. 

Also, one implementation question. This change means that is you make the `api/stream` request with Livepeer.com admin's token and without a `userId` specified, you will get ALL streams by ALL users returned, correct?

**Comment by darkdarkdragon** [comment]
@wohlner 
> Can I test this with a curl command or something in my terminal before it goes to stage? 

No. Only way is to start local dev env and test locally.

> This change means that is you make the api/stream request with Livepeer.com admin's token and without a userId specified, you will get ALL streams by ALL users returned, correct?

Yes. Same as it worked before.

**Comment by wohlner** [comment]
@darkdarkdragon in that case, I'm looking forward to testing when it is on .monster.

File: packages/www/hooks/use-api.tsx [hunk 1]
```diff
@@ -520,7 +520,7 @@ const makeContext = (state: ApiState, setState) => {
 
     async getStreams(userId): Promise<Array<Stream>> {
       const [res, streams] = await context.fetch(
-        `/stream?userId=${userId}&streamsonly=1`
+        `/stream?streamsonly=1`
```
Comment (iameli) [inline packages/www/hooks/use-api.tsx]: I think this should still have `userId` — I'm an admin but I still want my "streams" page to only show mine.

Comment (darkdarkdragon) [reply packages/www/hooks/use-api.tsx]: Sure! Missed that one 😢

**Pull request merged**